### ISSUE TITLE
Towards a GA release on PyPI

### ISSUE THREAD

**Issue by amotl** [ralight/mqttshark#4]
Hi there,

after giving some love to mqttwarn the other day, converging it from an ad hoc program conceived by @jpmens and @sumnerboy12 into a regular Python package, I would like to do the same with mqttshark.

It will need some additional project metadata (`pyproject.toml`), a refactoring of the `mqttshark` file into a module directory, so it will effectively most probably land at `mqttshark/core.py`, and probably a few other auxiliaries.

Let me know if you have any objections about it, or if you support that idea.

With kind regards,
Andreas.

**Comment by sumnerboy12** [comment]
I don't have any objections, although I do not use mqttshark so my vote should carry little weight!

**Comment by ralight** [comment]
As long as I can't tell any difference when I run it with any of the options then that's fine.

**Comment by amotl** [comment]
Yeah, it will effectively be the same, and will roughly work like this, from a user's perspective.

```
pip install --upgrade mqttshark
mqttshark --version
```
